tox.ini

```diff
@@ -1,2 +1,3 @@
 [flake8]
-max-line-length=99
\ No newline at end of file
+max-line-length=99
+extend-ignore=E741
```

Comment:
Is flake8 flagging out violations of this rule?

Reply:
Here is what I get once I enable that rule:

```console
$ ./dev/linter.sh
src/skmultiflow/evaluation/evaluate_prequential_delayed.py:349:17: E741 ambiguous variable name 'l'
src/skmultiflow/meta/oza_bagging.py:258:25: E741 ambiguous variable name 'l'
src/skmultiflow/meta/oza_bagging.py:270:13: E741 ambiguous variable name 'l'
src/skmultiflow/meta/online_smote_bagging.py:346:29: E741 ambiguous variable name 'l'
src/skmultiflow/meta/online_smote_bagging.py:362:13: E741 ambiguous variable name 'l'
src/skmultiflow/meta/online_boosting.py:339:29: E741 ambiguous variable name 'l'
src/skmultiflow/meta/online_boosting.py:356:13: E741 ambiguous variable name 'l'
src/skmultiflow/meta/online_adac2.py:363:29: E741 ambiguous variable name 'l'
src/skmultiflow/meta/online_adac2.py:379:13: E741 ambiguous variable name 'l'
src/skmultiflow/meta/online_under_over_bagging.py:320:29: E741 ambiguous variable name 'l'
src/skmultiflow/meta/online_under_over_bagging.py:336:13: E741 ambiguous variable name 'l'
src/skmultiflow/meta/online_csb2.py:356:29: E741 ambiguous variable name 'l'
src/skmultiflow/meta/online_csb2.py:373:13: E741 ambiguous variable name 'l'
src/skmultiflow/meta/online_rus_boost.py:385:29: E741 ambiguous variable name 'l'
src/skmultiflow/meta/online_rus_boost.py:402:13: E741 ambiguous variable name 'l'

```